### PR TITLE
add apdu to set and optionally freeze serial and extend length

### PR DESCRIPTION
**Pull request by swissbit-okaufmann**

### CODE REVIEW

File: src/net/cooperi/pivapplet/PivApplet.java [hunk 1]
```diff
@@ -139,6 +139,9 @@ public class PivApplet extends Applet
 	private static final byte INS_GET_SERIAL = (byte)0xf8;
 	private static final byte INS_GET_MDATA = (byte)0xf7;
 
+	/* SwissbitPIV extensions we support */
```
Comment (swissbit-csteuer) [inline src/net/cooperi/pivapplet/PivApplet.java]:
```suggestion
	/* SwissbitPIV extensions */
```

File: src/net/cooperi/pivapplet/PivApplet.java [hunk 2]
```diff
@@ -182,6 +185,8 @@ public class PivApplet extends Applet
 	private boolean pivPinIsDefault = true;
 	private boolean pukPinIsDefault = true;
 	private boolean mgmtKeyIsDefault = true;
+	private boolean serialIsFreezed = false;
```
Comment (swissbit-csteuer) [inline src/net/cooperi/pivapplet/PivApplet.java]:
```suggestion
	private boolean serialIsFrozen = false;
```

**Review by swissbit-csteuer — CHANGES_REQUESTED** [review]

File: src/net/cooperi/pivapplet/PivApplet.java (hunk 2 above)
Comment (swissbit-okaufmann) [reply src/net/cooperi/pivapplet/PivApplet.java]: renamed attribute

File: src/net/cooperi/pivapplet/PivApplet.java (hunk 1 above)
Comment (swissbit-okaufmann) [reply src/net/cooperi/pivapplet/PivApplet.java]: done

**Pull request merged**